index.html

```diff
@@ -7,14 +7,15 @@
         <link rel="icon" href="data:," />
         <script src="lib/matrix.js"></script>
         <script src="minimalTest.js"></script>
+        <script src="gameComm.js"></script>
         <link href="styles/style.css" rel="stylesheet" />
     </head>
     <body>
         <div id="wrapper">
             <h1 id="title"> Omnispace </h1>
             <div id="game">
                 <iframe id="game_frame" allow="cross-origin-isolated"
-                src = "https://telegram-space-l1c9paxke-igneous-labs.vercel.app/"> 
```

Comment:
can we revert this change? I don't actually have a local version running on my surface